assets/sass/_variables.scss

```diff
@@ -1,30 +1,30 @@
 // Page Properties
-$fonts: 'montserrat', Arial, Helvetica, sans-serif;
+$fonts: 'proxima nova', 'montserrat', Arial, Helvetica, sans-serif;
 $default-border-radius: 3px;
 $container-width: 400px;
 $container-min-width: 300px;
 $num-tabs: 5;
 
 // Font Size
-$font-size: 13px;
-$font-size-small: 12px;
-$font-size-header: 14px;
-$font-size-footer: 11px;
-$font-size-default: 14px;
+$font-size: 14px;
```

Comment:
@edburyenegren-okta just want to know what your thoughts to make this change. Does the look-n-feel work for you or who we shall double check?

Reply:
I defer to @vincentvoong-okta and @alexdahl-okta here until SIW starts using Odyssey.

At that point, I assume SIW and Okta Core will (separately) treat that as their baseline - ensuring the same styles in both places without SIW relying on Okta Core for code.